src/component/contents/DraftEditorLeaf.react.js

```diff
@@ -98,10 +98,10 @@ class DraftEditorLeaf extends React.Component {
       targetNode = child;
     } else if (child.tagName === 'BR') {
       targetNode = node;
-    } else if (child.firstChild && child.firstChild.nextSibling) {
+    } else if (child.firstChild.className === 'mozilla-fix') {
```

Comment:
If we are switching to named classes pick something more unique such as including the word draft. Also consider using the data property instead. Also why does this say mozilla when the code runs all the time?